packages/insomnia-app/app/main/window-utils.ts

```diff
@@ -407,6 +407,31 @@ export function createWindow() {
         type: 'separator',
       },
     );
+
+    /*
+    * This line has been introduced to make the Vim mode correctly work on MacOS
+    * Contrary to other platforms, if you preventDefault() a keypress event,
+    * no repeat events are fired for that key as it is being held down.
+    * This snippet, reintroduce the event firing disabling the Apple behaviour.
+    * The flag applies only at app-level (just for Insomnia),
+    * it won't influence other applications.
+    *
+    * Note [1]:
+    * if you've set ApplePressAndHoldEnabled at a global level
+    * (i.e. defaults write -g ApplePressAndHoldEnabled -bool true),
+    * this will override app-level preferences.
+    *
+    * Note [2]:
+    * The type definition for:
+    * setUserDefault(key: string, type: string, value: string): void
+    * is expecting a string for the 'value' field but not really...
+    * The implementation is expecting a primitive value, not a string.
+    * That's why the type-checking has been disabled in the next line.
+    * */
```

Comment:
```suggestion
    */
```

doing this mirrors how jsdoc comments terminate